packages/components/src/Persona.purs

```diff
@@ -47,7 +47,7 @@ getUser invalidateCache uuid token =
   where
     headers =
       { authorization
-      , cacheControl: fromMaybe mempty maybeCacheControl
+      , cacheControl: toNullable maybeCacheControl
```

Comment:
So this way the `cache-control` header doesn't appear?

Btw wasn't this introduced in a recent PR I reviewed?

Reply:
Oh no that was in `vetrina`

Reply:
It was yeah, it's a shared module though 😄

Reply:
Yes, if we set the `cacheControl` to `null`, it won't appear in the request